lib/simctl.js

```diff
@@ -366,15 +366,19 @@ async function createDevice (name, deviceTypeId, runtimeId, opts = {}) {
     log.warn(`Unable to find runtime for iOS '${runtimeId}'. Continuing`);
   }
 
-  if (util.compareVersions(await getCommandLineToolsVersion(), '>=', CMDLINE_TOOLS_FORMAT_CHANGED_VERSION)
-      || util.compareVersions(await getVersion(false), '>=', XCODE_FORMAT_CHANGED_VERSION)) {
-    // 1st comparison: getCommandLineToolsVersion
-    // Command line tool version is 10.2+, but xcode 10.1 can happen
-
+  const clangVersion = await getClangVersion();
+  // 1st comparison: clangVersion
+  // Command line tools version is 10.2+, but xcode 10.1 can happen
```

Comment:
Is this comment still necessary?

Reply:
yes, it is still useful to let people know that both xcode version and command line tools version matter